apps/webstore/src/app/pages/auth/signup/signup.component.html

```diff
@@ -49,17 +49,20 @@ <h1 class="page-title" i18n="@@signup">Sign up</h1>
                 >Password</label
             >
         </div>
-
+        <div class="info-message" i18n="@@info-password">
+            Password must contain: min 9 characters, at least one upper case
```

Comment:
The password minimum length is 6
```suggestion
            Password must contain: min 6 characters, at least one upper case
```